client/blocks/product-purchase-features-list/sell-online-paypal.jsx

```diff
@@ -0,0 +1,30 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PurchaseDetail from 'components/purchase-detail';
+
+export default localize( ( { translate } ) => {
+	return (
+		<div className="product-purchase-features-list__item">
+			<PurchaseDetail
+				buttonText={ translate( 'Start earning' ) }
+				description={ translate(
+					'Add a PayPal payment button and start selling your goods and services.'
+				) }
+				href={ 'https://en.support.wordpress.com/simple-payments/' }
```

Comment:
Should this point to https://jetpack.com/support/simple-payment-button/ for Jetpack sites instead?

Reply:
Oh, and one more thing: For string literal JSX props, we should use double quotes instead of curly braces plus single quotes, i.e.

```jsx
href="https://en.support.wordpress.com/simple-payments/"
```

(If we end up setting the `href` dynamically depending on whether we're on a JP or WP.com site, this is obviously a moot point.)

Reply:
> Should this point to https://jetpack.com/support/simple-payment-button/ for Jetpack sites instead?
hmmm... good point!
/cc @joanrho

Reply:
Great catch, @ockham! Yes to point the Jetpack sites to https://jetpack.com/support/simple-payment-button/  and WordPress.com sites to the https://en.support.wordpress.com/simple-payments/. Thanks!